shanoir-ng-front/src/app/studies/study-list/study-list.component.ts

```diff
@@ -261,7 +259,7 @@ export class StudyListComponent extends BrowserPaginEntityListComponent<Study> {
                     const text: string = 'You are a member of at least one study that needs you to accept its data user agreement. '
                         + 'Until you have agreed those terms you cannot access to any data from these studies. '
                         + 'Would you like to review those terms now?';
-                    const buttons = {ok: 'Yes, proceed to the signing page', cancel: 'Later'};
+                    const buttons = {yes: 'Yes, proceed to the signing page', cancel: 'Later'};
```

Comment:
Hi @julien-louis, I think this belongs to DUA. Is this something missing in today's develop? and you use this PR to bring it into prod? In case yes, it would be good to mention it in the PR "notes".

Reply:
I just changed the variable name from 'ok' to 'yes' because I added the option 'no' to use the confirmation component for the "do you want to test qc on a sample ? yes / no,all exams / cancel" choice.